src/System.Net.WebSockets.Client/tests/ClientWebSocketUnitTest.cs

```diff
@@ -64,8 +79,33 @@ public void ClientWebSocket_NoConnect_CloseOutputAsync_Ok()
         }
 
         [Fact]
-        public void ClientWebSocket_NoConnect_ReceiveAsync_Throws()
+        public void CloseAsync_CreateAndReceive_ThrowsInvalidOperationException()
         {
+            if (WebSocketHelper.ShouldSkipTestOSNotSupported(_output))
+            {
+                return;
+            }
+
+            var cws = new ClientWebSocket();
```

Comment:
I think we're slightly inconsistent on this point because the Http tests, for the most part, show proper dev pattern and have `Dispose` or `using` patterns. The exception is when we are explicitly testing for certain behavior with various patterns of using/not-using `Dispose`.

With regard to these tests, some of the tests are explicitly testing against different `Dispose` semantics, i.e. call methods after `Dispose`, etc.

I'll make a note to be more explicit about this. I think we should be more consistent.